graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/inputs/Extractor.java

```diff
@@ -124,7 +124,7 @@ public static Type fuzzyValueOf(String s) {
     public Extractor(MetricRegistry metricRegistry,
                      String id,
                      String title,
-                     int order,
+                     long order,
```

Comment:
Any reason to change the type to `long`? It's rather unlikely that we'll have more than 2147483647 extractors for an input.